lib/dashboard/repl.js

```diff
@@ -0,0 +1,37 @@
+const repl = require("repl");
+
+class REPL {
+  constructor(options) {
+    this.env = options.env;
+    this.plugins = options.plugins;
+  }
+
+  start(done) {
+    this.replServer = repl.start({
+      prompt: "Embark (" + this.env + ") > ",
+      useGlobal: true
+    });
+
+    this.replServer.on("exit", () => {
+      process.exit();
+    });
+
+    let self = this;
+    this.replServer.defineCommand('profile', {
```

Comment:
You needed to add a custom profile command? That's a problem, because user's can add their own console commands using a plugin and looks like they wouldn't work like this.

Reply:
would it be a possible solution to do an embark command,
e.g `.embark profile MyContract`

Otherwise do you have a suggestion?

Reply:
Now loading the Console and execute the same command